WEBPAGE/main.py

```
@@ -0,0 +1,911 @@
+        emailType = request.form['emailType']
+        subject = request.form['subject']
+        message_body = request.form['message']
+
+        logger.info(f"Email type: {emailType}, Subject: {subject}")
+        logger.info(f"Message body: {message_body}")
+
+        if emailType == "*":
+            emails = DB_interface.get_data("SELECT SchoolEmail FROM ACCOUNTS WHERE RoleID=0 AND (SchoolEmail IS NOT NULL)")
+            emails += DB_interface.get_data("SELECT HomeEmail FROM ACCOUNTS WHERE RoleID=0 AND (HomeEmail IS NOT NULL)")
+        else:
+            emails = DB_interface.get_data(f"SELECT {emailType} FROM ACCOUNTS WHERE RoleID=0 AND {emailType} IS NOT NULL")
```

Ruff (0.14.7)

Possible SQL injection vector through string-based query construction

(S608)